modules/system/models/File.php

```diff
@@ -80,7 +80,7 @@ public function getPublicPath()
             $uploadsPath .= '/protected';
         }
 
-        return Url::asset($uploadsPath) . '/';
+        return Storage::url($uploadsPath) . '/';
```

Comment:
@Teranode did you try my suggestion of not making any change to the filesystem config, removing the `uploadsPath` config call from this method and just doing `$this->getDisk()->url('/protected');` vs `$this->getDisk()->url('/public')`?

Reply:
that could would but i would also need to append the `uploads` name to it would that work for you @LukeTowers